lessons/operator/05/README.md

```diff
@@ -1,261 +1,148 @@
+Observability describes a quality of the system; a measure of how much visibility we have into its state and behaviors.
 
-**Example check structure:**
+The more you _observe_, the richer your knowledge is of the system.
+The richer that knowledge is, the better you can understand and reason about the system.
 
-```json
-{
-  "metadata": {
-    "labels": {},
-    "annotations": {}
-  },
-  "entity": {},
-  "check": {},
-  "metrics": {},
-  "id": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
-  "timestamp": 0123456789
-}
-```
+So what is it that we are observing?
+
+### System Transparency 
+From our vantage point looking at a dashboard, we can only observe what the system shares with us.
+In traditional software development this involved informational output in the form of logs.
+Later, some systems were instrumented to allow on-demand inspection.
+
+It's now commonly understood that there is great value in providing a lot of transparency into system state.
+We design systems to emit, or allow the inspection of, as much detail as we can.
+Our data processing and storage systems have evolved past the need to constrain or limit this information.
+
+The more of this data we have available, the better off we are when trying to understand unexpected system behaviors.
+
+### The Function of Time
+
+The modern model involves two categories of system information; data which is _emitted_, and data which is _queried_.
+Broadly, emitted data is referred to as _logs_ or _events_, and queried or inspected data is known as _metrics_ or _telemetry_.
+These differ primarily in their relationship to time.
+
+Log messages are emitted at irregular and unpredictable times. 
+They are emitted when _something happens_. 
+Metrics however are queried, either by _polling_ on an periodic basis, or _ad hoc_ when someone is inspecting the system.
+
+The unifying property of these two kinds of information is their _timestamp_.
+
+### Structured Data
+
+Where these two kinds of information differ though, is in structure.
+Logs are presented as text strings, often in a human language. 
+They are intended to be human-readable and have little to no structure.
+Metrics are often presented as measurements in numerical form. 
+They are intended to be machine-readable and are often structured as a key-value pair.
 
-A single Sensu Event payload may include one or more metric `points`, represented as a JSON object containing a `name`, `tags` (key/value pairs), `timestamp`, and `value` (always a float).
+For a long time these two kinds of information were handled separately.
+In Sensu, we have developed a way to bring them together in a unifying structure: the _event_.
+
+## Working With Events
+
+There is a constant stream of these _observations_ in different formats coming from every layer of our systems.
```

Comment:
Yeah, the sentence is trying to do too much at once. 
- constant stream
- observations in different formats
- every layer of our systems

Especially the last phrase is a bit awkward because it ends in a plural, when you're expecting the singular "a system" vs "our systems"... 

Leaving as is for now, because I can't think of a better way to phrase it quickly and deadline looms.